backend/api/db/crud/orientation_request.py

```diff
@@ -0,0 +1,22 @@
+import logging
+from uuid import UUID
+
+from asyncpg import Record
+from asyncpg.connection import Connection
+
+from api.db.models.orientation_request import OrientationRequest
+
+logger = logging.getLogger(__name__)
+
+
+async def get_orientation_request_by_id(
+    connection: Connection, orientation_request_id: UUID
+) -> OrientationRequest | None:
+    orientation_requests = await connection.fetch(
```

Comment:
```suggestion
    orientation_requests : list[Record] = await connection.fetch(
```
Typer orientation_requests en ajoutant l'import adéquat